_plugins/fetch_remote.rb

```diff
@@ -49,14 +49,6 @@ def pre_read(site)
         end
       end
 
-      Jekyll.logger.info "  Fixing up URLs in swagger files"
-      Dir.glob("./engine/api/*.yaml") do |file_name|
-        Jekyll.logger.info "    #{file_name}"
-        text = File.read(file_name)
-        replace = text.gsub!("https://docs.docker.com/", "")
```

Comment:
Ah yeah, replace should not strip the `/`

Are there other places where we do this?

Reply:
I was checking other pages too with hardcoded `docs.docker.com` which leads to unexpected behavior for review because it doesn't check the actual domain with htmlproofer. I have another branch that will fix that and remove hardcoded root url as you suggested in https://github.com/docker/docker.github.io/pull/14784#discussion_r878382327